pkg/solana/client/multinode_client.go

```diff
@@ -300,8 +300,40 @@ func (m *MultiNodeClient) GetInterceptedChainInfo() (latest, highestUserObservat
 	return m.latestChainInfo, m.highestUserObservations
 }
 
-func (m *MultiNodeClient) SendTransaction(ctx context.Context, tx *solana.Transaction) error {
-	// TODO: Use Transaction Sender
-	_, err := m.SendTx(ctx, tx)
-	return err
+type SendTxResult struct {
+	err   error
+	txErr error
+	code  mn.SendTxReturnCode
+	sig   solana.Signature
+}
+
+var _ mn.SendTxResult = (*SendTxResult)(nil)
+
+func NewSendTxResult(err error) *SendTxResult {
+	return &SendTxResult{
+		err: err,
```

Comment:
We need to classify the error, otherwise the mn.SendTxReturnCode is 0, which does not match any value

Reply:
Added the error classification. However, result.Error() should be checked and handled before result.TxError() or result.Code() are used.